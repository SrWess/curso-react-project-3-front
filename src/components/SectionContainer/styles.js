import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    width: 85%;
    margin: 0 auto;
    padding: ${theme.spacings.large};
  `}
`;
