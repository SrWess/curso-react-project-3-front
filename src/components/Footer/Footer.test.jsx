import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { Footer } from '.';

describe('<Footer />', () => {
  it('should render', () => {
    const { container } = renderTheme(
      <Footer html={'<h1>Test component</h1>'} />,
    );
    expect(
      screen.getByRole('heading', { name: 'Test component' }),
    ).toBeInTheDocument();
    expect(container).toMatchInlineSnapshot(`
      .c0 {
        text-align: center;
      }

      .c0 a {
        color: inherit;
        -webkit-text-decoration: none;
        text-decoration: none;
        font-size: 1.6rem;
      }

      .c1 {
        max-width: 1200rem;
        margin: 0 auto;
        padding: 3.2rem;
      }

      <div>
        <footer
          class="c0"
        >
          <div
            class="c1"
          >
            <div
              class=""
            >
              <h1>
                Test component
              </h1>
            </div>
          </div>
        </footer>
      </div>
    `);
  });
});
