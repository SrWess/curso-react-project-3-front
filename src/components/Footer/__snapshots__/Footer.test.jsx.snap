// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
.c0 {
  text-align: center;
}

.c0 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.6rem;
}

.c1 {
  max-width: 1200rem;
  margin: 0 auto;
  padding: 3.2rem;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class=""
      >
        Children
      </div>
    </div>
  </div>
</div>
`;
